Modules/_threadmodule.c

```diff
@@ -11,7 +11,7 @@
 #include "pycore_pylifecycle.h"
 #include "pycore_pystate.h"       // _PyThreadState_SetCurrent()
 #include "pycore_time.h"          // _PyTime_FromSeconds()
-#include "pycore_weakref.h"       // _PyWeakref_GET_REF()
+#include "pycore_weakref.h"       // _PyWeakref_GET_REF(), FT_CLEAR_WEAKREFS()
```

Comment:
There is no need to list all includes, one is enough:

```suggestion
#include "pycore_weakref.h"       // _PyWeakref_GET_REF()
```